applications/tari_console_wallet/src/ui/state/wallet_event_monitor.rs

```diff
@@ -213,6 +216,26 @@ impl WalletEventMonitor {
                             Err(broadcast::error::RecvError::Closed) => {}
                         }
                     },
+                    event = contacts_liveness_events.recv() => {
+                        match event {
+                            Ok(liveness_event) => {
+                                match liveness_event.deref() {
+                                    ContactsLivenessEvent::StatusUpdated(data) => {
+                                        trace!(target: LOG_TARGET,
+                                            "Contacts Liveness Service Callback Handler event 'StatusUpdated': {}",
```

Comment:
```suggestion
                                            "Contacts Liveness Service event 'StatusUpdated': {}",
```

We are in the console_wallet so callbacks are not really relavent